torch/_torch_docs.py

```diff
@@ -6672,18 +6672,30 @@ def merge_dicts(*dicts):
 If :attr:`some` is ``True``, then this function returns the thin (reduced) QR factorization.
 Otherwise, if :attr:`some` is ``False``, this function returns the complete QR factorization.
 
+.. warning:: ``torch.qr`` is deprecated. Please use ``torch.linalg.`` :meth:`~torch.linalg.qr`
```

Comment:
:func:, not :meth:, since torch.linalg.qr is a function

Reply:
This is has some funky formatting:

<img width="720" alt="Screen Shot 2020-12-24 at 9 01 38 PM" src="https://user-images.githubusercontent.com/38511765/103120127-42039080-462b-11eb-8fd4-af482da50af6.png">

Can we make the entire name a reference?

Reply:
partially fixed by a5570d7797,

> This is has some funky formatting:

I know, I have already mentioned it in a previous comment: https://github.com/pytorch/pytorch/pull/47764#discussion_r538629779

The problem is that `` :meth:`~torch.linalg.qr` `` is rendered as simply `qr` by sphinx: what I did is a workaround but I agree it should be properly solved eventually.

Reply:
I agree this is a reasonable workaround.

Reply:
Late here, this may already be solved, but -- does using ```:func:`torch.linalg.qr` ``` (i.e., removing the `~`) not render the link the way you want?

The `~` [tells Sphinx](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-syntax) to only render the simple object name, rather than the fully-qualified name that (I think) you want here.

Reply:
Never too late. Thanks for fixing this, @bskinn!